packages/api-sdk/src/medical/models/demographics.ts

```diff
@@ -8,6 +8,11 @@ import {
   stripNonNumericChars,
 } from "../../shared";
 
+export const generalTypes = ["passport", "ssn", "medicare"] as const;
```

Comment:
Can we add a bit more detail to `generalTypes`? This is quite generic.

Also, no need to use `Type` on the name. We've been using `schema` or nothing, and for the actual type, no suffix.

Maybe something like `generalPersonalIdentifiers`?

Reply:
Done.